sdk/servicebus/azure-servicebus/azure/servicebus/aio/async_message.py

```diff
@@ -22,14 +22,14 @@ class Message(message.Message):
     :type encoding: str
 
     Example:
-        .. literalinclude:: ../examples/test_examples.py
+        .. literalinclude:: ../samples/sync_samples/test_examples.py
```

Comment:
This is in "async_message" - not sure if this should be an async sample?

Reply:
Interestingly, sync vs async doesn't matter in this case, the sample in question (complex_message) would be the same in both cases; and async doesn't have a parallel implementation (other than queue_client_send which is subtlely different.)

I can replicate the former into the async world if needed but it would be only to ensure these paths align, not because the sample itself changes.

Let me know where your preference lies in that; I'm ambivalent but if the paths would in some way confuse someone reading the docs, which would absolutely push me the other direction.

Reply:
Sounds fine :)